lib/ruby_lsp/requests/semantic_highlighting.rb

```diff
@@ -52,6 +52,12 @@ def run
         @encoder.encode(@tokens)
       end
 
+      def visit_def(node)
```

Comment:
Can we do singleton and endless definition methods too?

Reply:
https://github.com/Shopify/ruby-lsp/pull/134

Reply:
Thank you!